### DISCUSSION TITLE
Trying to pre-install extensions via CLI gives an error

### DISCUSSION THREAD

**Discussion by helloguille** [coder/code-server#6035] [Q&A]
I have the following error when trying to install extensions via command line.

I did it following the instructions in https://github.com/microsoft/vscode-remote-release/issues/1718

root@instance# code-server --install-extension bmewburn.vscode-intelephense-client
error: Found argument '--install-extension' which wasn't expected, or isn't valid in this context

**Comment by code-asher** [comment]
It looks like this might be Microsoft's `code-server` script rather than ours.  Are you using VS Code remote or this project?  If the latter how are you installing code-server?
